src/MessageHandler/ZipDatasetFilesHandler.php

```diff
@@ -96,7 +96,7 @@ public function __invoke(ZipDatasetFiles $zipDatasetFiles)
         $datasetSubmissionId = $zipDatasetFiles->getDatasetSubmissionId();
         $datasetSubmission = $this->entityManager->getRepository(DatasetSubmission::class)->find($datasetSubmissionId);
         $filesInfo = $this->fileRepository->getFileNameAndPath($fileIds);
-        $destinationPath = $this->downloadDirectory . DIRECTORY_SEPARATOR . $datasetSubmission->getDataset()->getUdi() . '.zip';
+        $destinationPath = $this->downloadDirectory . DIRECTORY_SEPARATOR . str_replace(':', '.', $datasetSubmission->getDataset()->getUdi()) . '.zip';
```

Comment:
This can stay tho...

Reply:
William wanted the zipFileName also without the colon